crates/net/rpc-builder/src/lib.rs

```diff
@@ -197,23 +254,51 @@ impl RpcServer {
     ///
     /// This returns an [RpcServerHandle] that's connected to the server task(s) until the server is
     /// stopped or the [RpcServerHandle] is dropped.
-    pub fn start(self, _methods: impl Into<Methods>) -> Result<RpcServerHandle, RpcError> {
-        todo!()
+    pub async fn start(
+        self,
+        methods: impl Into<Methods> + Copy,
```

Comment:
this won't work. because there's no Into<Methods> type that is also Copy.

I'm ok with leaving this entire function as a `todo!` because I think we need to figure out all the config settings first.

Reply:
alright. will make changes.